LanguageFeatures/Patterns/Exhaustiveness/exhaustiveness_map_A01_t01.dart

```diff
@@ -11,7 +11,7 @@
 // SharedOptions=--enable-experiment=patterns
 
 String test1(Map<bool, bool> m) =>
-  switch (m) {
+  switch (m) { // Empty map is not matched
```

Comment:
Great, thanks!

Reply:
We should probably get into a habit of adding a comment whenever we have a non-trivial discussion. If we do this all the time for many days, and we have to discuss "what should this code do?" then surely it will be helpful for a future reader of the code to get a hint about the contents and outcomes of those discussions.

Reply:
Agreed!